common/lib/symmath/symmath/test_formula.py

```diff
@@ -19,6 +19,7 @@ class FormulaTest(unittest.TestCase):
     mathml_end = '</mstyle></math>'
 
     def setUp(self):
+        super(FormulaTest, self).setUp()
```

Comment:
Maybe all tests should be checked for a `super().setUp()` call? With a:
`find . -name test_\*.py -exec grep -C1 "def setUp(" {} \; -print` - or the like?